assets/js/blocks/customer-account/block.json

```diff
@@ -0,0 +1,33 @@
+{
+	"name": "woocommerce/customer-account",
+	"version": "1.0.0",
+	"title": "Customer account",
+	"description": "A block that allows your customers to log in and out of their accounts in your store.",
+	"category": "woocommerce",
+	"keywords": [ "WooCommerce" ],
+	"supports": {
+		"align": true,
+		"color": {
+		  "__experimentalDefaultControls": {
```

Comment:
Just curious, why is `__experimentalDefaultControls` necessary and what's the difference with simply having:
```JSON
		"color": {
			"text": true
		},
```

Reply:
You are right, I think it's not needed.